documentation/docs/decentralized_identifiers/overview.md

```diff
@@ -9,68 +9,69 @@ keywords:
+DIDs become more interesting in combination with Verifiable Credentials, which will be covered in a later section. Essentially, Verifiable Credentials (VCs) are signed statements by trusted third parties about a certain identity. The signer, or Issuer, is referenced by the DID and so is the subject, often called the Holder. The Holder controls a copy of this statement and share it with other parties, the Verifiers, that can verify the statement and check which party made the statement, without having to ask the Issuer. Instead, they can verify the signature of the Issuer by checking the Issuers DID Document. This whole setup puts Holders back in control over their own data, but also makes the data much more trustworthy as it has become verifiable.
 
-## Why use IOTA Identity over other implementations?
+## Why Use IOTA Identity Over Other Implementations?
 
-IOTA Identity is a framework to implement Self-Sovereign Identities on IOTA. Inherently, IOTA provides some unique features that have a major impact on the usability of the framework.
+IOTA Identity is a framework to implement SSI on IOTA. Inherently, IOTA provides unique features that have a major impact on the usability of the framework.
 
 ### Feeless
 
-IOTA is a feeless Distributed Ledger Technology, which means that messages can immutably be stored inside the Tangle at no cost, nor a requirement of holding any cryptocurrency tokens. That means that SSI applications can directly deploy towards the main network without any problems, as compared to most other SSI solutions running on a test network or having cryptocurrency requirements. This doesn't just make IOTA Identity have predictable costs and prevent issues around cryptocurrency holding taxes and legislation, it also makes it a fair network as anyone would be able to create one or more identities at no cost. The wealth of someone is irrelevant, making it the most inclusive SSI solution.
+IOTA is a feeless Distributed Ledger Technology, which means that messages can immutably be stored inside the Tangle at no cost, nor a requirement of holding any cryptocurrency tokens. That means that SSI applications can directly deploy towards the main network without any problems, as compared to most other SSI solutions running on a test network or having cryptocurrency requirements. So not only does it give IOTA Identity predictable costs, but it also turns it into a fair network.
```

Comment:
I feel like we're losing a bit of the explanation here.

Reply:
Reversed